lib/team_result_parser.rb

```diff
@@ -130,19 +135,22 @@ def alltime_goals_per_team #helper method
     goals_hash
   end
   
-  def best_offense(teams)
+  def best_offense
     best_offense = nil
+    @league_parser.list_teams(@team_path)
+    teams = @league_parser.get_team_list
```

Comment:
This was a great idea in order to ensure that we did not need an argument for this method!